vividus-plugin-web-app/src/test/java/org/vividus/selenium/WebDriverTypeTests.java

```diff
@@ -60,6 +62,14 @@
 @PowerMockIgnore("io.github.lukehutch.fastclasspathscanner.classpath.*")
 public class WebDriverTypeTests
 {
+    private static final String OPTION_VALUE = "iPhone 8";
+
```

Comment:
Does empty lines between fields necessary  ?

Reply:
It is style supported by eclipse field extraction; Any concerns about the formatting ??

Reply:
Nope,  just interest. :-)